src/pages/landing/components/RegisterSection.tsx

```diff
@@ -0,0 +1,168 @@
+              {...register("companyName", { required: true })}
+            />
+            <TextField
+              className="mb-6"
+              required fullWidth variant="standard"
+              label="Contact email" type="email"
+              error={errors.contactEmail !== undefined}
+              helperText={errors.contactEmail !== undefined ? 'The field value is invalid' : undefined}
+              {...register("contactEmail", { required: true, pattern: /^[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-_]+\.[A-Za-z]{2,64}$/i })}
+            />
+            <TextField
+              className="mb-6"
+              required fullWidth variant="standard"
+              label="Email" type="email"
+              error={errors.email !== undefined}
+              helperText={errors.email !== undefined ? 'The field value is invalid' : undefined}
+              {...register("email", { required: true, pattern: /^[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-_]+\.[A-Za-z]{2,64}$/i })}
+            />
+            <TextField
+              className="mb-6"
+              required fullWidth variant="standard"
+              label="Full name" type="text"
+              error={errors.fullName !== undefined}
+              helperText={errors.fullName !== undefined ? 'The field value is invalid' : undefined}
+              {...register("fullName", { required: true })}
+            />
+            <TextField
+              className="mb-6"
+              fullWidth variant="standard"
+              label="Twitter" type="text"
+              error={errors.twitter !== undefined}
+              helperText={errors.twitter !== undefined ? 'The field value is invalid' : undefined}
+              {...register("twitter", { pattern: /^[A-Za-z0-9_]{1,15}$/i })}
+            />
+            <TextField
+              className="mb-6"
+              fullWidth variant="standard"
+              label="LinkedIn" type="url"
+              error={errors.linkedin !== undefined}
+              helperText={errors.linkedin !== undefined ? 'The field value is invalid' : undefined}
+              {...register("linkedin", { pattern: /^(http(s)?:\/\/)?([\w]+\.)?linkedin\.com\/(pub|in|profile|company)\/([a-zA-Z0-9_-]+)$/i })}
+            />
+            <TextField
+              className="mb-6"
+              fullWidth variant="standard"
+              label="Website" type="url"
+              error={errors.website !== undefined}
+              helperText={errors.website !== undefined ? 'The field value is invalid' : undefined}
+              {...register("website", { pattern: /^(https?:\/\/)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,255}\.[a-z]{2,6}(\b([-a-zA-Z0-9@:%_\+.~#()?&\/\/=]*))?$/i })}
+            />
+            <Button
+              type="submit" variant="contained" size="large"
+              className="mt-8 py-3 px-14 normal-case bg-main"
+              disabled={total <= 0 || processing}
+            >
+              Pay (₳{total.toFixed(2)})
```

Comment:
Wrong price in ada displayed. I see 0.50A (roughly 2Ada, I guess). The conversion should be in the other order.
Also the price should be fetched from the tier table.

Reply:
Ok, from the `/tiers` the price I get is `"usdPrice": 2` and from the `/ada-usd-price` I get `0.246953`
That's why I got `2 * 0.246953 = 0.493906 A`, but I'm seeing that the price in the subscription is `8098707`
So I don't know how to get the value for the payment button without having the subscription. What calculation do you use?

Reply:
The price you get from the tier is in USD and the ada-usd-price is the price of 1 Ada in USD. So you have to who 2/Ada-use-price, which is 2 / 0.246953 = 8.098707A. Which should be displayed, rounded up after 2 decimals, so here display 8.10A.

The price requested by the wallet is in Lovelaces and 1Ada = 1,000,000 Lovelaces so in this case 8,098,707 Lovelaces